### PR TITLE
Deprecate ShrinkWrapping - Part III

### PR DESCRIPTION
**Pull request by fjeremic**
Remove some last bits and pieces of Shrink Wrapping code in the stack
walker.

Issue: https://github.com/eclipse/omr/issues/2107

Signed-off-by: Filip Jeremic <fjeremic@ca.ibm.com>

### CODE REVIEW

**Comment by fjeremic** [comment]
Changes are rather simple but wouldn't hurt a review from either @gacholio or @keithc-ca given changes in their area of expertise. FYI this removes the last bit of code related to shrink wrapping in OpenJ9 (AFAIK).

**Comment by gacholio** [comment]
Looks reasonable to me - I don't think you should edit the DDR artifacts. @keithc-ca ?

**Comment by keithc-ca** [comment]
The changes to files in `DDR_Artifacts` have no effect on OpenJ9 builds, they're only relevant to the legacy tooling used in IBM builds and they're not needed (or wanted) there.

**Comment by keithc-ca** [comment]
I don't know what 'shrink wrapping' means in this context, but if _any_ JVM based on OpenJ9 exists that might generate a core file to be examined, the support for interpreting such core files should remain.

**Comment by fjeremic** [comment]
> The changes to files in `DDR_Artifacts` have no effect on OpenJ9 builds, they're only relevant to the legacy tooling used in IBM builds and they're not needed (or wanted) there.

Ok, I'll revert the spec changes then. On another note if these files have no effect on OpenJ9 builds can we migrate them to the IBM internal repo that cares about them? Why pollute the open source repo if the files are meaningless? Should we open up an issue for this? If you can specify which files can be migrated I will gladly do the work.

**Comment by fjeremic** [comment]
Jenkins test sanity all JDK8

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java [hunk 1]
```diff
@@ -451,43 +448,23 @@ public void jitAddSpilledRegisters(WalkState walkState, VoidPointer stackMap) th
...
-						savedGPRs = registerSaveDescription.bitAnd(new UDATA(31));
-						saveOffset = registerSaveDescription.rightShift(11);
-						lowestRegister = new UDATA(32).sub(savedGPRs);
-					}
+					/*
+						* see PPCLinkage for a description of the RSD 
+						* the save offset is located from bits 18-32 
+						* so first mask it off to get the bit vector 
+						* corresponding to the saved GPRS
+						*/
```
Comment (keithc-ca) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java]: Could you please fix the indentation of this comment so is consistent?

Comment (fjeremic) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/MethodMetaData.java]: Fixed in 19913fe

**Comment by 0xdaryl** [comment]
Jenkins test sanity all JDK11

Launching again because the record of whether the previous PR build was successful was lost on the last push, and I did not see the results.

**Pull request merged**